src/conf_mode/interfaces-bridge.py

```diff
@@ -346,6 +347,13 @@ def apply(bridge):
 
         # add interfaces to bridge
         for member in bridge['member']:
+            # flushes address of only children of Interfaces class
```

Comment:
This code is used multiple time, should it be moved within ifconfig ?

Reply:
Can you be more specific? We can't move the `if intf in Section.interfaces`, the other 2 klass lines maybe, but I don't know exactly where it would make sense.